eradiate/scenes/biosphere/tests/test_biosphere_leaf_cloud.py

```diff
@@ -1,5 +1,6 @@
 import os
 import tempfile
+from fileinput import filename
```

Comment:
How do you use this? The docs for this method are very short and I can't make sense of them like this :sweat_smile:

Reply:
Thanks for spotting that, I think it sneaked while I was cleaning up and sorting imports.